src/Components/Atoms/buttons/hamburger-button/hamburger.js

```diff
@@ -0,0 +1,63 @@
+import './hamburger.scss';
+import React, { useEffect, useRef, useState } from 'react';
+import Menu from 'src/Components/Molecules/menu/menu';
+import SubMenu from 'src/Components/Molecules/sub-menu.js/sub-menu';
+
+const Hamburger = () => {
+    const [isActive, setActive] = useState(false);
```

Comment:
does isActive means is open?

Reply:
isActive is open(hamburger X), while(X) is visible it counts as active.